### DISCUSSION TITLE
QuantumultX 使用问题：能否在不代理(只使用direct)的情况下使用规则去广告，定时JS等功能

### DISCUSSION THREAD

**Discussion by chinanoahli** [blackmatrix7/ios_rule_script#1118] [Q&A]
QuantumultX能否做到只使用 direct 通信以达到在关闭代理通信的情况下，保留去广告、定时JS任务之类的功能？

**Comment by bautistaclay6027** [comment]
> QuantumultX能否做到只使用 direct 通信以达到在关闭代理通信的情况下，保留去广告、定时JS任务之类的功能？

可以的，你把配置里的那些策略组的子策略都改成direct就可以了，去广告是需要reject的，所以你全局直连不行，改策略组的仍然可以去广告。

**Reply by chinanoahli** [reply]
有示例吗，策略组的配置我一直没搞懂怎么写

**Reply by bautistaclay6027** [reply]
要啥示例啊，就是分流。分流的那些策略组除了去广告的都给改成Direct直连，兜底的final也给改成Direct就可以了。
